### ISSUE TITLE
sometimes stop (square) button is shown for a stopped stream

### ISSUE THREAD

**Issue by 532910** [y20k/transistor#321]
I dunno how to reproduce this ):

**Comment by 532910** [comment]
I found a way to reproduce, but not sure if this is the same issue:

Steps:
1. Start playback
2. Kill app with long back (Enable `Kill app back button` in Developer options before)
3. Open transistor again. The stop button will be displayed without the option to start playback again.

**Comment by y20k** [comment]
Thx. I have experienced that behavior, too. I happens when the app crashes while playing back radio - or is restarted via Android Studio

**Comment by y20k** [comment]
Thanks for the description to reproduce the bug and for filing it. It's just a "papercut" issue, but I am glad to fix it. Next update will contain the patch.

**Comment by 532910** [comment]
Another, more common, step sequence, though it doesn't replicate every time:

1. Start playback
2. Switch to home screen
3. Stop playback from notification
4. Clear the notification
5. Open transistor

**Comment by Jonas-itquirks** [comment]
Hm I don't know if this is related, I have the problem that after closing the app and notification controls after some time the notification controls pop up again with the square showing.
The notification controls are not removable in this state and no control works, no radio is playing, the app itself is not visibly running.
I have to start the app, start a stream to make the notification controls work again after this I can remove them. They don't close with the app, I am on 4.0.10.

**Comment by y20k** [comment]
Hi @Jonas-itquirks Thanks for reporting!! Your bug sounds like #297. If so, please re-open 297 and add a description how reliably reproduce the bug. The OS version would be of interest, too. Cheers

**Comment by Jonas-itquirks** [comment]
Yes it sounds exactly like what is mentioned in #297 and #274, I did not search closed issues before :/ .

I am on Android 10 realme x2 pro, software version RMX1931_11_C.34 with security patch: Nov, 2020/Dec, 2020

I have removed data and uninstalled the app and reinstalled from github directly (I previously installed from Playstore). I will open #297 again if it starts happening again and repost the info.
Thanks.
